concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1064,7 +1064,44 @@ jobs:
                 done > config/config.sh
 
                 CF_MANIFEST=cf-manifest/cf-manifest.yml $(pwd)/paas-cf/concourse/scripts/extract_quota_settings.rb config
+
+                cd config
+                CF_MANIFEST=../cf-manifest/cf-manifest.yml $(pwd)/../paas-cf/concourse/scripts/extract_security_groups.rb
+
       - aggregate:
+        - task: update-default-security-groups
```

Comment:
The `update-default-security-groups` task does not cover the situation where we would want to remove a default security group (not implying said situation will ever occur).